internal/rbd/rbd_attach_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestParseMapOptions(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name              string
+		mapOption         string
+		expectKrbdOptions string
+		expectNbdOptions  string
+		expectErr         string
+	}{
+		{
+			name:              "with old format",
+			mapOption:         "kOp1,kOp2",
+			expectKrbdOptions: "kOp1,kOp2",
+			expectNbdOptions:  "",
+			expectErr:         "",
+		},
+		{
+			name:              "with new format",
+			mapOption:         "krbd:kOp1,kOp2;nbd:nOp1,nOp2",
+			expectKrbdOptions: "kOp1,kOp2",
+			expectNbdOptions:  "nOp1,nOp2",
+			expectErr:         "",
+		},
+		{
+			name:              "without krbd: label",
```

Comment:
as the target function name it key, it would have been better to put it as `without krbd: key`, but not an issue as well

Reply:
@pkalever are you making above change?

Reply:
Sure if you need them, I thought it was just cosmic. Let me know if you need them please, I'm happy to make a respin, but that will retrigger all the tests :-(

Reply:
thats fine..